File: src/entity/ticketo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entity;

/**
 *
 * @author firas
 */
public class ticketo {
    private int numTicketO ;
    private String  Logement ;
    private String  Restauration ;
    private float prix ;

    public ticketo() {
    }

    public ticketo(int numTicketO, String Logement, String Restauration, float prix) {
        this.numTicketO = numTicketO;
        this.Logement = Logement;
        this.Restauration = Restauration;
        this.prix = prix;
    }

    public int getNumTicketO() {
        return numTicketO;
    }

    public void setNumTicketO(int numTicketO) {
        this.numTicketO = numTicketO;
    }

    public String getLogement() {
        return Logement;
    }

    public void setLogement(String Logement) {
        this.Logement = Logement;
    }

    public String getRestauration() {
        return Restauration;
    }

    public void setRestauration(String Restauration) {
        this.Restauration = Restauration;
    }

    public float getPrix() {
        return prix;
    }

    public void setPrix(float prix) {
        this.prix = prix;
    }

    @Override
    public String toString() {
        return "ticketo{" + "numTicketO=" + numTicketO + ", Logement=" + Logement + ", Restauration=" + Restauration + ", prix=" + prix + '}';
    }

    

    
   
    
    

    
    
}

File: Downloads/integration web/src/Entity/Ticketo.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * Ticketo
 *
 * @ORM\Table(name="ticketo")
 * @ORM\Entity
 */
class Ticketo
{
    /**
     * @var int
     *
     * @ORM\Column(name="numTicketO", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $numticketo;

    /**
     * @var string
     *
     * @ORM\Column(name="Logement", type="string", length=255, nullable=false)
     */
    private $logement;

    /**
     * @var string
     *
     * @ORM\Column(name="Restauration", type="string", length=255, nullable=false)
     */
    private $restauration;

    /**
     * @var float
     * @Assert\NotNull(message="Indiquer un prix")
     * @Assert\Positive (message="Le prix doit etre positif")
     * @ORM\Column(name="Prix", type="float", precision=10, scale=0, nullable=false)
     */
    private $prix;

    public function getNumticketo(): ?int
    {
        return $this->numticketo;
    }

    public function getLogement(): ?string
    {
        return $this->logement;
    }

    public function setLogement(string $logement): self
    {
        $this->logement = $logement;

        return $this;
    }

    public function getRestauration(): ?string
    {
        return $this->restauration;
    }

    public function setRestauration(string $restauration): self
    {
        $this->restauration = $restauration;

        return $this;
    }

    public function getPrix(): ?float
    {
        return $this->prix;
    }

    public function setPrix(float $prix): self
    {
        $this->prix = $prix;

        return $this;
    }


}
